modules/intro/pages/applications.adoc

```diff
@@ -40,10 +40,107 @@ Duplicating an application is useful for quickly trying out new options while st
 
 === Share an application
 
-Click on btn:[Share] in the three-dot menu to bring up the Share window.
+==== Share via login
+* Click on btn:[Share] in the three-dot menu to bring up the Share window.
+
+* Select one of the listed users.
 
 image::share-application.png[]
 
+==== Share via access token
+
+If you want to share your widget with a user who does not have an Insights login, you can send them an access token.
+
+==== Creating database account
```

Comment:
(1) The heading levels probably need to be adjusted.  "Creating data account" (H4) and subsequent headers are intended to be subsections of "Share via access token", which is also H4. Stepping back, the v2.9.2 page has 5 sections which are H2,H3,H3,H3, and H3. The last one is "Share an application." I think those 5 should actually be all on the top level: H2,H2,H2,H2,H2.  "Share via login" and "Share via access token" should be H3. Then these new subsections can be H4.

Reply:
(2) Phrasing and consistency of phrasing of section headings:
Most of these sections are one of the steps the user should perform to share an application.  For step-by-step instructions, I think we should use the imperative voice: "Create a database account" / "Configure nginx", etc.  (But " Query permissions required to write/run queries" is not a step; it is informational.)

Reply:
This was changed in latest commit.